.github/workflows/release.yaml

```diff
@@ -10,8 +10,10 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      dockerhub_image_name: docker.io/bitnami/sealed-secrets-controller
-      ghcr_image_name: ghcr.io/bitnami-labs/sealed-secrets-controller
+      dockerhub: docker.io/bitnami
+      ghcr: ghcr.io/bitnami-labs
+      controller_image_name: sealed-secrets-controller
+      kubeseal_image_name: kubeseal
```

Comment:
Please, change it to sealed-secrets-kubeseal. To follow the same format as the controller.

Reply:
Done, thanks ^^